src/stylesheets/components/_contact-panel.scss

```diff
@@ -0,0 +1,24 @@
+.app-c-contact-panel {
+  @include govuk-responsive-margin($govuk-spacing-responsive-6, "top");
+  @include govuk-responsive-padding($govuk-spacing-responsive-3);
+  background-color: $govuk-blue;
+}
+
+.app-c-contact-panel--heading {
+  @extend .govuk-heading-m;
+}
+
+.app-c-contact-panel--body {
+  @extend .govuk-body;
+  margin-bottom: 0;
```

Comment:
If we don't want the margins from the body class or the text colour, I think we should probably just be including `govuk-font-regular-19` here?